vscode/microsoft-kiota/CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to Kiota extension
+
+Thanks for your interest in contributing to Kiota! We welcome contributions from everyone, regardless of skill level or experience. Here are some guidelines to help you get started:
+
+## Getting Started
+
+To get started, you'll need to have the following tools installed:
+
+- [.NET SDK 8.0](https://get.dot.net/8)
+- [Visual Studio Code](https://code.visualstudio.com/)
+
+
+
+## Building the project
+
+```sh
+# On Linux/MacOS
+sudo dotnet workload restore
+# On Windows, run the following command in an elevated prompt
+dotnet workload restore
+# then
+dotnet restore
+dotnet build
```

Comment:
dotnet build should do the restore automatically. Maybe it may be helpful to reduce the commands then?